src/components/UI/ButtonStyled/RadioButtonstyled.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+export const Fieldset = styled.fieldset`
+	display: flex;
+	border: none;
+	height: 3rem;
+	display: flex;
+	align-items: center;
+	margin-bottom: 2rem;
+	margin-left: 10vw;
+	padding-left: 2rem;
+	width: 80vw;
+	background: linear-gradient(
+		45deg,
+		rgba(59, 173, 227, 1) 0%,
+		rgba(87, 111, 230, 1) 25%,
+		rgba(152, 68, 183, 1) 51%,
+		rgba(255, 53, 127, 1) 100%
+	);
```

Comment:
If you use this gradient multiple times, you can put it into a css variable in your globalstyles.